cmd/tsbs_generate_queries/main.go

```diff
@@ -41,12 +42,17 @@ var useCaseMatrix = map[string]map[string]utils.QueryFillerMaker{
 	},
 }
 
+const(
+	defaultWriteSize  = 4 << 20 // 4 MB
```

Comment:
since its the only `const`, you can just do the one-liner form

Reply:
Agree, made it as a one-liner